core/drivers/gic.c

```diff
@@ -208,7 +209,8 @@ void gic_init(struct gic_data *gd, vaddr_t gicc_base __maybe_unused,
 #endif
 }
 
-static int gic_dt_get_irq(const uint32_t *properties, int len)
+static int gic_dt_get_irq(const uint32_t *properties, int len,
+			  uint32_t *type __unused, uint32_t *prio __unused)
```

Comment:
Would it make sense to set `type` and `prio` to `IRQ_TYPE_NONE` and `0` respectively?
The point would be to make sure that we don't use garbage. It may help simplifying other functions too if they can depend on these always being set on a successful return.

Reply:
Yes, that was the point of the init in the generic dt_get_irq_type_prio.